api/swagger.yaml

```diff
@@ -7958,6 +7958,9 @@ paths:
           description: |
             A JSON encoded value of the filters (a `map[string][]string`) to process on the secrets list. Available filters:
 
+            - `id=<secret id>`
+            - `label=<key> or label=<key>=value`
+            - `name=<secret name>`
             - `names=<secret name>`
```

Comment:
Ah, I see, it was already used behind the scenes https://github.com/docker/docker/pull/30810/files#diff-74117112d8788c60f76ddd5fdb6e62b4R30